draft-patton-cfrg-vdaf.md

```diff
@@ -856,7 +864,9 @@ A concrete `Prg` implements the following class method:
 
 Each `Prg` has two derived class methods. The first is used to derive a fresh
 seed from an existing one. The second is used to compute a sequence of
-pseudorandom field elements froma pseudorandom byte string.
+pseudorandom field elements froma pseudorandom byte string. This method used is
```

Comment:
... The second one generates a sequence of field elements from a pseudorandom byte string. Such a generation is based on ...

Reply:
Done